### PR TITLE
 Removed case which prevented a retryAfter for unique jobs

### PR DESCRIPTION
**Pull request by qxh5696**
PR to remove the case which prevents a retryAfter for unique jobs, which we decided to do in [this github issue](https://github.com/Expensify/Expensify/issues/84654) and concluded in [this github conversation](https://github.com/Expensify/Expensify/issues/84654#issuecomment-424258389).

### CODE REVIEW

File: test/tests/jobs/CreateJobTest.cpp [hunk 1]
```diff
@@ -280,14 +279,6 @@ struct CreateJobTest : tpunit::TestFixture {
         tester->executeWaitVerifyContent(command, "402 Malformed retryAfter");
     }
 
-    void retryUnique() {
```
Comment (iwiznia) [inline test/tests/jobs/CreateJobTest.cpp]: Why remove it instead of updating it to show the new behavior?

Comment (qxh5696) [reply test/tests/jobs/CreateJobTest.cpp]: Lol so I knew if I did it one way you'd ask for the other so I just went with the easiest solution first. Will update.

Comment (iwiznia) [reply test/tests/jobs/CreateJobTest.cpp]: lol, not really true, I am always up for having more tests, not less 😄

**Comment by qxh5696** [comment]
Hmm, this CI build failure had nothing to do with my changes. Is there a way to restart this build?

**Comment by qxh5696** [comment]
Of course, I wasn't logged in 🤦‍♂️

**Comment by qxh5696** [comment]
bump @iwiznia

**Pull request merged**